gtsam/geometry/tests/testPinholeCamera.cpp

```diff
@@ -22,11 +22,13 @@
 #include <gtsam/base/Testable.h>
 #include <gtsam/base/numericalDerivative.h>
 
+#include <boost/bind/bind.hpp>
 #include <CppUnitLite/TestHarness.h>
 
 #include <cmath>
 #include <iostream>
 
+using namespace boost::placeholders;
```

Comment:
This is an example @acxz

Reply:
It is used right here: https://github.com/borglab/gtsam/pull/796/files/650e432f52f58505eba8c8f6a918f03e733a7acc#diff-55c680128d3251f02bf918b24cbe80c0b30fe8ff56ecda7f187d6bb3f5fdbde2R70

Reply:
Link doesn't really work that well, line 70 of the new file.

Reply:
Which new file? The diff for this file doesn't show anything else.

Reply:
Sorry about that wasn't clear. Line 68 of the current `gtsam/geometry/tests/testPinholeCamera.cpp` or Line 70 of the same file with my commit applied.

https://github.com/acxz/gtsam/blob/650e432f52f58505eba8c8f6a918f03e733a7acc/gtsam/geometry/tests/testPinholeCamera.cpp#L70